admin/worker/delete_unsued_assets.go

```diff
@@ -0,0 +1,80 @@
+package worker
+
+import (
+	"context"
+	"errors"
+	"net/url"
+	"strings"
+	"time"
+
+	"cloud.google.com/go/storage"
+	"go.uber.org/zap"
+	"golang.org/x/sync/errgroup"
+)
+
+const _defaultPageSize = 20
+
+func (w *Worker) deleteUnusedAssets(ctx context.Context) error {
+	// We skip unused assets created in last 15 minutes to prevent race condition
+	// where somebody just created an asset but is yet to use it
+	createdBefore := time.Now().Add(-15 * time.Minute)
```

Comment:
This is cutting it a little close (clocks can diverge, or we may change the upstream value and forget to change here). Maybe something like 6 hours?